### ISSUE TITLE
accessibility affected by  Sfdatagrid virtualization, turn it off as an option?

### ISSUE THREAD

**Issue by therenca** [syncfusion/flutter-widgets#2186]
### Bug description

Do we have a way of turning off virtualization of cells, to allow for accessibility. As it is, columns not within the viewport are hidden from accessibility thus one is unable to pan all the way to the end of columns when using tools such as talkback

### Steps to reproduce

setup the example data table for employes, enable talkback from device and try navigating

### Code sample

accessbility issue, reproducable from running the example project for Sfdatagrid

### Screenshots or Video

accessibility issue

### Stack Traces

no stack trace

### On which target platforms have you observed this bug?

Android, iOS

### Flutter Doctor output

accessibility issue

**Comment by abineshPalanisamy** [comment]
Hi @therenca ,

In SfDataGrid, columns and rows outside of the viewport are not created due to the use of virtualization, which optimizes performance. The accessibility feature in the Flutter DataGrid allows you to read the cell contents by tapping on any visible cell. Since virtualization is used, columns and rows outside the visible area are not pre-created; they are dynamically built as you scroll. 
 
However, you can still navigate and read cells that are out of view by scrolling the grid to bring the content into the viewport. **You can scroll the DataGrid both vertically and horizontally using touch gestures, such as dragging with two fingers, ensuring all cells are accessible to screen readers like TalkBack**.
 
UG link - [Accessibility in Flutter DataGrid](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelp.syncfusion.com%2Fflutter%2Fdatagrid%2Faccessibility&data=05%7C02%7Cabinesh.palanisamy%40syncfusion.com%7C246f36b366e44e8266c108dd0f7585a9%7C77f1fe12b04949198c509fb41e5bb63b%7C0%7C0%7C638683719598855051%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=RzcFH6UxVxRCejj5BTnkdggqvhy7v4qHTMuq7y4BYas%3D&reserved=0).

If there are any misunderstandings regarding your requirements or if your needs differ, please provide specific and clear details about the issue for a more thorough analysis. Additionally, kindly share a video recording demonstrating the issue for better clarity. This additional information will help us address your request comprehensively and provide an appropriate solution.
 
Regards,
Abinesh P

**Comment by ashok-kuvaraja** [comment]
Hi @therenca,

We suspect that the reported issue has been resolved at your end. Hence, we are closing this issue. If you need any further assistance, please reopen this. We are always happy to help.

Regards,
Ashok K
